docs/product-manuals/clients/java-client-examples/index.md

```diff
@@ -18,19 +18,19 @@ Import the Maven project in the `samples` directory into your IDE to start hacki
 
 ## Process
 
-- [Deploy a process](process-deploy.md)
-- [Create a process instance](process-instance-create.md)
-- [Create non-blocking process instances](process-instance-create-nonblocking.md)
+- [Deploy a Process](process-deploy.md)
+- [Create a Process Instance](process-instance-create.md)
+- [Create Non-Blocking Process Instances](process-instance-create-nonblocking.md)
 - [Create a Process Instance with Results](process-instance-create-with-result.md)
```

Comment:
Should we match how this is capitalized in the sidebar and title? These are inconsistent with your update.

Reply:
You're correct here -- adjusting so they match their given titles in the sidebar.